recipe/meta.yaml

```diff
@@ -15,29 +15,22 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
     - python
-    - hdf5 1.10.1.*
-    - libnetcdf 4.*
+    - libnetcdf
     - lapack >=3.4.2
     - clapack >=3.2.1
```

Comment:
You can remove `lapack` and `clapack` and replace with openblas.

Reply:
Done